calcs/leewayAngle.js

```diff
@@ -0,0 +1,17 @@
+// calculation source: https://arvelgentry.jimdo.com/app/download/9157993883/Arvel+Gentry+-+Sailboat_Performance_Testing_Techniques.pdf?t=1485748085
+module.exports = function (app, plugin) {
+  return {
+    group: 'heading',
+    optionKey: 'leewayAngle',
+    title: 'Leeway (based on heading and COG)',
+    derivedFrom: [
+      'navigation.headingMagnetic',
+      'navigation.courseOverGroundTrue'
+    ],
+    calculator: function (hdg, cog) {
+      var leewayAngle = Math.abs(hdg - cog)
+      // app.debug("leeway angle: " + leewayAngle);
+      return [{ path: 'performance.leeway', value: leewayAngle }]
```

Comment:
Should be navigation.leewayAngle

https://signalk.org/specification/1.5.0/doc/otherBranches.html#aircraftregexpnavigationleewayangle